api/client.yaml

```
@@ -1513,6 +1513,248 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/moov-io/base/master/api/common.yaml#/components/schemas/Error'
+  /customers/{customerID}/representatives:
+    post:
+      tags: [Customers]
+      summary: Add customer representative
```

```suggestion
      summary: Add Customer Representative
```

In this [PR](https://github.com/moov-io/customers/pull/271/files) Adam capitalized all summaries.